src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class FigureSupplier extends ColorSupplier {

```

Comment:
The `FigureSupplier` class should not extend `ColorSupplier`. Instead, you should create an instance of `ColorSupplier` within the `FigureSupplier` class and use it to get random colors.